mmcv/transforms/processing.py

```diff
@@ -438,8 +438,8 @@ class CenterCrop(BaseTransform):
 
     Args:
         crop_size (Union[int, Tuple[int, int]]):  Expected size after cropping
-            with the format of (h, w). If set to an integer, then cropping
-            height and width are equal to this integer.
+            with the format of (w, h). If set to an integer, then cropping
+            width and height are equal to this integer.
```

Comment:
what is the motivation for change the order of `crop_size`

Reply:
Cause we use (w, h) order in `Resize` and `Pad`.

Reply:
> 

Involve @mzr1996 to take a look. If it is true, we can make it a convention and update both the user and design documentation.

Reply:
Right, that's the design in the design doc.